### ISSUE TITLE
Conflict with nvim-ts-rainbow

### ISSUE THREAD

**Issue by ecosse3** [DNLHC/glance.nvim#55]
Hi. I have below error when I have enabled `nvim-ts-rainbow` plugin for treesitter and going to reference via Glance. I don't know if it's error related to Glance or nvim-ts-rainbow.

Error:

```vim
Error executing lua callback: Vim:E315: ml_get: Invalid lnum: 1
stack traceback:
        [C]: in function 'bufload'
        vim/_editor.lua: in function 'region'
        /usr/local/share/nvim/runtime/lua/vim/highlight.lua:30: in function 'range'
        ...share/nvim/lazy/nvim-ts-rainbow/lua/rainbow/internal.lua:86: in function 'update_range'
        ...share/nvim/lazy/nvim-ts-rainbow/lua/rainbow/internal.lua:194: in function 'cb'
        ...l/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:780: in function '_do_callback'
        ...l/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:220: in function 'invalidate'
        ...l/share/nvim/runtime/lua/vim/treesitter/languagetree.lua:921: in function '_on_detach'
        /usr/local/share/nvim/runtime/lua/vim/treesitter.lua:73: in function </usr/local/share/nvim/runtime/lua/vim/treesitter.lua:69>
        [C]: in function 'nvim_buf_delete'
        ...local/share/nvim/lazy/glance.nvim/lua/glance/preview.lua:177: in function 'close'
        ...i/.local/share/nvim/lazy/glance.nvim/lua/glance/init.lua:502: in function 'close'
        ...i/.local/share/nvim/lazy/glance.nvim/lua/glance/init.lua:443: in function 'jump'
        ...i/.local/share/nvim/lazy/glance.nvim/lua/glance/init.lua:300: in function <...i/.local/share/nvim/lazy/glance.nvim/lua/glance/init.lua:299>
```

**Comment by DNLHC** [comment]
Hey, i tried [nvim-ts-rainbow](https://github.com/p00f/nvim-ts-rainbow) and it works fine for me.
I noticed this plugin is no longer maintained, is this the one you're using?
What is your neovim version?

upd. Was able to reproduce, the error sometimes occurs when glance window is closing.

**Comment by ecosse3** [comment]
@DNLHC I was using `mrjones2014/nvim-ts-rainbow`. I have changed it to `hiphish/rainbow-delimiters.nvim` for now and looks like this issue is solved. I won't use `nvim-ts-rainbow` since it's deprecated.

I've had another issue from `lewis6991/gitsigns.nvim` while working with `Glance`, but I've changed the loading event from `BufReadPre` to `BufRead` in `lazy.nvim` for gitsigns and the issue has been solved.
